cpp/Ice/greeter/Server.cpp

```diff
@@ -26,6 +26,7 @@ main(int argc, char* argv[])
 
     // Start dispatching requests.
     adapter->activate();
+    cout << "Listening on port 4061..." << endl;
```

Comment:
This is new too: I find it clearer when the server outputs something at startup.